### PR TITLE
add cgo flag back

### PR DESCRIPTION
**Pull request by coleenquadros**
https://issues.redhat.com/browse/ACM-12309

### CODE REVIEW

**Review by douglascamata — APPROVED** [review]
/lgtm

File: Dockerfile.prow [hunk 1]
```diff
@@ -6,7 +6,7 @@ WORKDIR /workspace
 COPY . .
 
 RUN go mod vendor
-RUN go install github.com/prometheus/promu@v0.17.0 && make common-build
+RUN go install github.com/prometheus/promu@v0.17.0 && promu build -v --cgo --prefix ./
```
Comment (saswatamcode) [inline Dockerfile.prow:10]: Same here, why not just add the flag to the build target instead of separately using promu

File: Makefile.common [hunk 2]
```diff
@@ -196,6 +196,10 @@ common-unused:
 
 .PHONY: common-build
 common-build: promu
+ifneq ($(shell which promu), $(PROMU))
+	@echo "\nswitching promu path: $(PROMU) to use: $(shell which promu)\n"
+	$(eval PROMU = $(shell which promu))
+endif
```
Comment (saswatamcode) [inline Makefile.common]: Maybe I'm missing something, why do we need this

File: Dockerfile.prow (hunk 1 above)
Comment (coleenquadros) [reply Dockerfile.prow:10]: CI / Build Alertmanager for common architectures (1) (pull_request)  these builds fail when I add the CGO flags

**Pull request merged**